noggin/themes/fas/templates/main.html

```diff
@@ -96,13 +96,15 @@ <h2 class="display-4">{{_("Fedora Accounts")}}</h2>
 {# an optional macro defining an element to show for editing group details #}
 {# typically, it used to link to a ticket tracker for an admin to edit a group #}
 {% macro edit_group_details() %}
-    {% if current_user_is_sponsor and sponsor_form %}
-        <div class="col-auto ml-auto">
-            <a class="btn btn-sm btn-outline-primary" href="https://pagure.io/fedora-infrastructure/new_issue" title="{{_('file a Fedora Infra ticket to change the details or sponsors of this group')}}">
-                {{_("Request Change of Details")}}
-            </a>
-        </div>
-    {% endif %}
+    <a tabindex="0"  
```

Comment:
To avoid the `href=javascript` stuff, you can use a `button` tag with `btn-link` in the classes.